File: C++/First Steps in Coding - Basics of C++/First steps/Concatenate data.cpp
#include <iostream>
#include <cstdlib>
using namespace std;
int main() {
	string FirstName, LastName, town; int age;
  	cin>>FirstName; cin>>LastName; cin>>age; cin>>town;
  	cout<<"You are "<<FirstName<<" "<<LastName<<", a "<<age<<"-years old person from "<<town<<".";
  return 0;
}

File: Go/First Steps Basics/Concatenate Data.go
package main

import "fmt"

func main() {
	var name, lastName, town string
	var age int
	fmt.Scanln(&name)
	fmt.Scanln(&lastName)
	fmt.Scanln(&age)
	fmt.Scanln(&town)
	message := fmt.Sprintf("You are %v %v, a %v-years old person from %v.", name, lastName, age, town)
	fmt.Print(message)
}
